src/dataprotection/azext_dataprotection/manual/aaz_operations/recovery_point.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.aaz import AAZStrArg
+from azext_dataprotection.aaz.latest.dataprotection.recovery_point import List as _RecoveryPointList
+from ..helpers import validate_recovery_point_datetime_format
+
+
+class RecoveryPointList(_RecoveryPointList):
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+        _args_schema = cls._args_schema
+        _args_schema.start_time = AAZStrArg(
+            options=["--start-time"],
+            help="Specify the start date time in UTC (yyyy-mm-ddTHH:MM:SS)",
+        )
```

Comment:
You can use `AAZDateTimeArg` 
https://github.com/Azure/azure-cli/blob/82358037c5af4a43c46facfcff054100a730c678/src/azure-cli-core/azure/cli/core/aaz/_arg.py#LL258C7-L258C21

It will format the input value in iso datetime. 
https://github.com/Azure/azure-cli/blob/82358037c5af4a43c46facfcff054100a730c678/src/azure-cli-core/azure/cli/core/aaz/_arg_fmt.py#L200-L205

Reply:
Tried using that. But `AAZDateTimeArg` formats the string to yyyy-mm-ddTHH:MM:SS.000Z and also converts the time to UTC considering input as local time. The API call only considers yyyy-mm-ddTHH:MM:SS.0000000Z as a valid format and user input is expected in UTC.